### PR TITLE
[DOC] Normalization notebook (WIP)

### LINKED ISSUE #135: [DOC] Produce notebook showing how to normalize and standardize columns using pyjanitor
I like to add a normalize and standardize function to pyjanitor. I guess we can do this with transform function, but I like the readability of the code over a lambda function to make it work. Like normalize_columns() or standardize_columns(), since we already have expand_columns() (aka. dummy codes) and max_min_scale(). This will put preprocessing into the hands of more beginners, even though I have nothing against sklearn. I will say, doing a .fit() call on preprocessing bothers me, and I dont know why? What are your thoughts?

### PR DESCRIPTION
**Pull request by catherinedevlin**
Prepared the notebook attached to #135 for incorporation into docs.

**WARNING:** When I `build html` locally, the embedded plots throw warnings

    /home/catherine/proj/pyjanitor/docs/notebooks/normalize.ipynb:: WARNING: image file not readable: notebooks/../_build/doctrees/nbsphinx/notebooks_normalize_12_1.png

and do not render in the Sphinx docs.  We may need to hold this PR until that's resolved.

# PR Description

@jcvall attached a PDF of a notebook to #135 demonstrating normalization and standardization with pyjanitor.  This PR uses that code and adds explanatory text.

**This PR resolves #135.**

1. [x ] PR in from a fork off your branch. Do not PR from `<your_username>`:master, but rather from `<your_username>`:<branch_name>.

2. [x] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

## Quick Check

To do a very quick check that everything is correct, follow these steps below:

- [x] Run the command `make check` from pyjanitor's top-level directory. This will automatically run:
    - black formatting
    - pycodestyle checking
    - running the test suite
    - docs build
    
Once done, please check off the check-box above.
    
If `make check` does not work for you, you can execute the commands listed in the Makefile individually.

## Documentation Changes

If you are adding documentation changes, please ensure the following:

- [x] Build the docs locally.
- [ ] View the docs to check that it renders correctly.  SEE note above!

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

**Comment by ericmjl** [comment]
@catherinedevlin I tried pulling down your PR and building the docs, but I ran into many more errors than I could debug.

For the time being, I'm going to label this PR as WIP, so that the WIP bot prevents me from hitting the big green button accidentally.

Also, tagging here @ricky-lim and @Eidhagen and @kessj, who worked a ton on the docs at the sprint. Hopefully putting a few minds together may help - but no pressure on the three of you to solve this!

**Comment by ericmjl** [comment]
@catherinedevlin after wrestling with the docs all of today, I'm realizing that I'm okay just merging in and fixing the missing image issue at a later time.
